spec/models/pawn_spec.rb

```diff
@@ -1,5 +1,137 @@
+        result = black_pawn.valid_move?('D', 2)
+        expect(result).to be false
+      end
+
+      it "white pawn can move 2 spaces" do
+        result = white_pawn.valid_move?('C', 5)
+        expect(result).to be true
+      end
+
+      it "white pawn can move 1 space" do
+        result = white_pawn.valid_move?('C', 6)
+        expect(result).to be true
+      end
+
+      it "white pawn can't move 3 spaces" do
+        result = white_pawn.valid_move?('C', 4)
+        expect(result).to be false
+      end
+
+      it "white pawn can't move backwards" do
+        result = white_pawn.valid_move?('C', 8)
+        expect(result).to be false
+      end
+
+      it "white pawn can't move horizontal" do
+        result = white_pawn.valid_move?('B', 7)
+        expect(result).to be false
+      end
+    end
+
+    context "not first move" do
+      let(:black_pawn) { build(:pawn, color: "black", x: 'C', y: 4) }
+      let(:white_pawn) { build(:pawn, color: "white", x: 'B', y: 6) }
+
+      it "black pawn can move 1 space" do
+        result = black_pawn.valid_move?('C', 5)
+        expect(result).to be true
+      end
+
+      it "black pawn can't move 2 spaces" do
+        result = black_pawn.valid_move?('C', 6)
+        expect(result).to be false
+      end
+
+      it "black pawn can't move backwards" do
+        result = black_pawn.valid_move?('C', 2)
+        expect(result).to be false
+      end
+
+      it "black pawn can't move horizontal" do
+        result = black_pawn.valid_move?('B', 4)
+        expect(result).to be false
+      end
+
+      it "white pawn can move 1 space" do
+        result = white_pawn.valid_move?('B', 5)
+        expect(result).to be true
+      end
+
+      it "white pawn can't move 2 spaces" do
+        result = white_pawn.valid_move?('B', 4)
+        expect(result).to be false
+      end
+
+      it "white pawn can't move backwards" do
+        result = white_pawn.valid_move?('B', 7)
+        expect(result).to be false
+      end
+
+      it "white pawn can't move horizontal" do
+        result = white_pawn.valid_move?('C', 6)
+        expect(result).to be false
+      end
+    end
+
+    context "capture" do
+      let(:black_pawn) { build(:pawn, color: "black") }
+      let(:white_pawn) { build(:pawn, color: "white") }
+
+      it "can capture one square diagonally" do
+        black_pawn.update(x: 'F', y: 3)
+        white_pawn.update(x: 'G', y: 4)
+
+        result = black_pawn.valid_move?('G', 4)
+        expect(result).to be true
+      end
+
+      it "can't capture vertically" do
+        black_pawn.update(x: 'F', y: 3)
+        white_pawn.update(x: 'F', y: 4)
+
+        result = black_pawn.valid_move?('F', 4)
+        expect(result).to be false
+      end
+
+      it "can't capture horizontally" do
+        black_pawn.update(x: 'F', y: 3)
+        white_pawn.update(x: 'E', y: 3)
+
+        result = black_pawn.valid_move?('E', 3)
+        expect(result).to be false
+      end
```

Comment:
tests are excellent. great use of `context` and you've got great coverage of all the different possible moves. :smile: